Modules/TreasureHunt/Presenters/TreasureHuntPresenter.php

```diff
@@ -31,14 +31,20 @@ public function createComponentIntroForm()
     {
         $form = $this->registerFormFactory->create();
         $form->onSuccess[] = function (Form $form, $values) {
+            // $values come in trimmed :)
             $nick = $values['nick'];
             $pass = implode('-', $values['pass']);
 
             $submitter = $form->isSubmitted();
             if ($submitter == $form['register']) {
-                $result = $this->userManager->register($nick, $pass);
-                if (!$result) {
-                    $this->flashMessage("Adresát $nick si již zásilku převzal");
+                if ($nick == "") {
+                    $this->flashMessage("Pro registraci zadejte uživatel. jméno");
```

Comment:
Uvědomil jsem si, že formuláře umí zobrazovat chyby a tedy bychom se mohli prozatím vyhnout flash zprávám mimo komponenty.

Můžeš prosím změnit použití `$this->flashMessage('...')` na `$form->addError('...', false)`?